src/sentry/models/outbox.py

```diff
@@ -367,6 +360,19 @@ def find_regions_for_user(user_id: int) -> Set[str]:
         }
 
 
+def find_regions_for_user(user_id: int) -> Set[str]:
+    org_ids: Set[int]
```

Comment:
For control silo models, we often need to map identifiers to region names.  This refactoring makes it possible to go from both user or raw org_ids to find the related regions.

Reply:
Makes sense to me. Publishing outbox messages on per-org basis is something we'll need for integrations and sentryapps.